src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    /**
+     *Produces a printable string representation of the object
+     */
+    String getPrintableString();
```

Comment:
Good job having a header comment for this method. You may want to have a header comment for the interface, as well.